packages/calling/calling/src/CallingClient/registration/types.ts

```diff
@@ -1,11 +1,53 @@
-import {WebexRequestPayload} from '../../common/types';
+import {LogContext} from 'Logger/types';
+import {METRIC_EVENT, METRIC_TYPE, REG_ACTION} from '../metrics/types';
+import {IDeviceInfo, MobiusStatus, WebexRequestPayload} from '../../common/types';
+import {CallingClientError} from '../../Errors';
 
 export type Header = {
   [key: string]: string;
 };
 
+export type restoreRegistrationCallBack = (
+  restoreData: IDeviceInfo,
+  loggerContext: LogContext
+) => Promise<boolean>;
+
 export interface IRegistrationClient {
-  createDevice: (url: string) => Promise<WebexRequestPayload>;
-  deleteDevice: (url: string, deviceId: string, deviceUrl: string) => Promise<WebexRequestPayload>;
+  postRegistration: (url: string) => Promise<WebexRequestPayload>;
+  deleteRegistration: (
```

Comment:
These no longer have to be public right?

Reply:
Removed it.